internal/noderesourcetopology/equality.go

```diff
@@ -80,3 +88,35 @@ func EqualResourceInfo(resInfoA, resInfoB nrtv1alpha2.ResourceInfo) (bool, error
 	}
 	return true, nil
 }
+
+func EqualResourceInfoWithDeviation(resInfoA, resInfoB nrtv1alpha2.ResourceInfo) (bool, error) {
+	/*
+		tests that involve reboot show a difference in the memory capacity on numa nodes, e.g memory capacity
+		 is reallocated across numa nodes upon startup. This is a a kernel behavior which we have no control
+		 on thus we want to work it around. The tests failed when comapring initial (pre-reboot) NRT with the
+		 final (post reboot) NRT, and finds out a difference of 53760000 of memory (=52500Ki ~=52Mi).
+		 For those case, we allow a deviation of 52Mi when comparing the memory topologies.
+	*/
+	dev, _ := resource.ParseQuantity("54525952") //52 Mi
```

Comment:
How did we come up with this value? The commit message and the PR says that we allow a deviation of 1Mi but we seem to be allowing a deviation of roughly 52Mi here.

Reply:
good catch, I need to update the commit message. After testing it turned out that 1Mi is not enough to cover the difference, and usually the difference we get is 53760000 which equal to ~51 Mi units of memory, I rounded that diff up and made it 52Mi. I'll fix the PR and the commit msg. sorry for the confusion.

Reply:
fixed, thanks